From c8faaa704d95e509c5bae7813fdb81e6f3344b88 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 31 May 2023 14:15:18 +0200
Subject: [PATCH 0283/1141] arm64: dts: st: add goodix pinmux on
 stm32mp25-pinctrl.dtsi

Define pin muxing to set interrupt line as input with pull down

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I96dd1dbb0162edbb93f0d1f929f6acd2f1c71644
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/309764
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi b/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
index 260c5bf1bbf3..aeb0b7f004b7 100644
--- a/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
@@ -136,6 +136,13 @@ pins4 {
 		};
 	};
 
+	goodix_pins_a: goodix-0 {
+		pins {
+			pinmux = <STM32_PINMUX('I', 13, GPIO)>;
+			bias-pull-down;
+		};
+	};
+
 	i2c2_pins_a: i2c2-0 {
 		pins {
 			pinmux = <STM32_PINMUX('B', 5, AF9)>, /* I2C2_SCL */
-- 
2.39.2

